src/Features/Core/Portable/Shared/TestHooks/Legacy/AggregateAsynchronousOperationListener.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.Shared.TestHooks
 {
+    [Obsolete("use IAsynchronousOperationListenerProvider")]
```

Comment:
:memo: This is the type of situation where dotnet/csharplang#384 would provide better assurances regarding the usefulness of the message.

Reply:
Consider using the overload to make it an error.

Reply:
aren't that be too intrusive? @minestarks

Reply:
@minestarks okay checking in?